src/import/FSHImporter.ts

```diff
@@ -126,6 +153,44 @@ export class FSHImporter extends FSHVisitor {
     });
   }
 
+  visitInstance(ctx: pc.InstanceContext) {
+    const instance = new Instance(ctx.SEQUENCE().getText())
+      .withLocation(this.extractStartStop(ctx))
+      .withFile(this.file);
+    this.parseInstance(instance, ctx.instanceMetadata(), ctx.fixedValueRule());
+    this.doc.instances.set(instance.name, instance);
+  }
+
+  private parseInstance(
+    instance: Instance,
+    metaCtx: pc.InstanceMetadataContext[] = [],
+    ruleCtx: pc.FixedValueRuleContext[] = []
+  ): void {
+    const seenPairs: Map<InstanceMetadataKey, string> = new Map();
+    metaCtx
+      .map(instanceMetadata => this.visitInstanceMetadata(instanceMetadata))
+      .forEach(pair => {
+        if (seenPairs.has(pair.key)) {
+          logger.error(
+            `Metadata field '${pair.key}' already declared with value '${seenPairs.get(pair.key)}'.`
+          );
+          return;
+        }
+        seenPairs.set(pair.key, pair.value);
+        if (pair.key === InstanceMetadataKey.InstanceOf) {
+          instance.instanceOf = pair.value;
+        } else if (pair.key === InstanceMetadataKey.Title) {
+          instance.title = pair.value;
+        }
+      });
+    if (!instance.instanceOf) {
+      throw new RequiredMetadataError('InstanceOf', 'Instance');
```

Comment:
If I'm following the code path correctly, I think this error will bubble all the way to the top and ultimately halt execution of the whole import process.  I think we want to catch this error in `visitInstance` and then log it.

Reply:
That makes sense. As for updating the test associated with this (`FSHImporter.Instance.Test.ts:44`), would it be fine to just check that the result has no instances?

Reply:
Yes, I think so -- although I suppose you _could_ also use your fancy jest spy to see if an error was logged.  But we do actually have that functionality (check for logged errors) as a separate story in our backlog.  So it could wait for that instead.  Up to you.  But either way, at least check that no instance is imported.